frontend/src/main/components/BasicCourseSearch/CourseByInstructorSearchForm.js

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+import { Form, Button, Container, Row, Col } from "react-bootstrap";
+import { quarterRange } from "main/utils/quarterUtilities";
+import { useSystemInfo } from "main/utils/systemInfo";
+import SingleQuarterDropdown from "../Quarters/SingleQuarterDropdown";
+
+const CourseByInstructorSearchForm = ({ fetchJSON }) => {
+
+  const { data: systemInfo } = useSystemInfo();
+
+  // Stryker disable OptionalChaining
+  const startQtr = systemInfo?.startQtrYYYYQ || "20211";
+  const endQtr = systemInfo?.endQtrYYYYQ || "20214";
+  // Stryker enable OptionalChaining
+
+  const quarters = quarterRange(startQtr, endQtr);
+
+  // Stryker disable all : not sure how to test/mock local storage
+  const localStartQuarter = localStorage.getItem("CourseByInstructorSearch.StartQuarter");
+  const localEndQuarter = localStorage.getItem("CourseByInstructorSearch.EndQuarter");
+  const localInstructor = localStorage.getItem("CourseByInstructorSearch.Instructor");
+
```

Comment:
Right here: we need:

// Stryker enable all

Reply:
`// Stryker enable all` didn't work but `// Stryker restore all` did. I also moved it down a few lines.

Reply:
To not move it down I think requires a much larger change to the SingleCourseDropdown since `quarter` is an unused parameter.